packages/react/src/hooks/useWagmiPassiveHook.ts

```diff
@@ -16,6 +18,23 @@ export function useWagmiPassiveHook<T>({
 }: UsePassiveHookParams<T>): T {
     const [browserFocused, setBrowserFocused] = useState(true);
 
+    const { data: blockNumber } = useBlockNumber({
```

Comment:
Shouldn't this only be enabled if the `watch` prop is true? Doesn't this always rerun the hook?

Reply:
This is enabled based on the condition below:

```ts
enabled: params.query?.enabled && browserFocused
```

It shouldn't run if it's disabled or the browser is not focused.

Reply:
Yes, but this doesn't need to run when the `wagmi` hook is enabled, but when `watch` is true too. Otherwise by default all hooks will be in watch mode.

Reply:
But that's the point of the `useWagmiPassiveHook`, it's a wrapper around the hooks that needs to be in watch mode, but with the browser focused logic added. Maybe I missed your point.

Reply:
Is it only for watch-based hooks?

Reply:
Yes, it's to avoid generating calls for the watch hooks when the browser is not focused.